### DISCUSSION TITLE
Creating a map given camera poses

### DISCUSSION THREAD

**Discussion by mattiasmar** [introlab/rtabmap#1085] [Q&A] [answered]
Can RTabMap create a new map given images with associated known camera poses?

**Comment by matlabbe** [comment, marked as answer]
Yes, with rtabmap standalone, you can provide an Odometry file in Source panel.
![image](https://github.com/introlab/rtabmap/assets/2319645/29209244-c1a8-4c37-8afe-15f1221671f7)

With rtabmap_ros, start rtabmap node and provide external odometry topic.
